src/main/java/com/google/sps/servlets/ServletUtils.java

```diff
@@ -21,21 +21,47 @@
 import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
 import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
 import com.google.cloud.secretmanager.v1.SecretVersionName;
+import com.google.sps.data.Election;
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.util.HashSet;
 import java.util.Optional;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import org.json.JSONObject;
 
 public class ServletUtils {
 
   private static final Logger logger = Logger.getLogger(ServletUtils.class.getName());
 
+  /**
+   * Get the value of a query parameter to an HTTP request.
+   *
+   * @param request the HTTP request to be serialized
+   * @param response the HTTP response to publish any error messages
+   * @param inputName the query parameter key
+   * @return Optional container that contains either the parameter or null
+   */
+  public static Optional<String> getRequestParam(
```

Comment:
You don't need to do it in this PR yet (although usually we would require it as such), but we should really start adding unit tests soon. Usually we have one file per file under test. So soemthing like ServletUtilsTest.java.

Reply:
+1 to this. This PR is being merged to the other PR I have open, since it's mostly refactoring and not really adding functionality. I'll be adding tests for ServletUtils to the other PR (sorry if this is confusing)

Reply:
ACK